tests/test_basic_s3.py

```diff
@@ -254,11 +270,19 @@ async def test_get_object_stream_wrapper(
     await create_object('foobarbaz', body='body contents')
     response = await s3_client.get_object(Bucket=bucket_name, Key='foobarbaz')
     body = response['Body']
-    chunk1 = await body.read(1)
-    chunk2 = await body.read()
+    if httpx and isinstance(body, httpx.Response):
+        # httpx does not support `.aread(1)`
+        byte_iterator = body.aiter_raw(1)
+        chunk1 = await byte_iterator.__anext__()
+        chunk2 = b""
+        async for b in byte_iterator:
+            chunk2 += b
+    else:
+        chunk1 = await body.read(1)
+        chunk2 = await body.read()
```

Comment:
it does, but you can't use `aread`, you need to use the other methods I mentioned in another file